packages/platform-sdk-profiles/src/wallets/wallet.ts

```diff
@@ -169,7 +169,7 @@ export class Wallet implements ReadWriteWallet {
 	}
 
 	public convertedBalance(): BigNumber {
-		if (this.network().isTest()) {
+		if (!this.network().isLive()) {
```

Comment:
Why is this necessary? If the `isTest` check doesn't work then `isTest` is bugged.

Reply:
not necessary at all, changing back.